packages/web/components/cards/validator-squad-card.tsx

```diff
@@ -0,0 +1,120 @@
+import { Staking } from "@keplr-wallet/stores";
+import React from "react";
+import { useTranslation } from "react-multi-lang";
+
+import { Button } from "~/components/buttons";
+import OsmoverseCard from "~/components/cards/osmoverse-card";
+import { useStore } from "~/stores";
+
+const maxVisibleValidators = 8;
+
+export const ValidatorSquadCard: React.FC<{
+  setShowValidatorModal: (val: boolean) => void;
+  validators?: Staking.Validator[];
+  usersValidatorsMap?: Map<string, Staking.Delegation>;
+}> = ({ setShowValidatorModal, validators, usersValidatorsMap }) => {
+  const t = useTranslation();
+  const { chainStore, queriesStore } = useStore();
+  const { chainId } = chainStore.osmosis;
+  const queries = queriesStore.get(chainId);
+
+  const queryValidators = queries.cosmos.queryValidators.getQueryStatus(
+    Staking.BondStatus.Bonded
+  );
+
+  let validatorBlock = (
+    <div className="flex flex-row space-x-2">
+      {Array(maxVisibleValidators)
+        .fill(0)
+        .map((_, index) => (
+          <AvatarIcon key={index} />
+        ))}
+    </div>
+  );
+
+  const myValidators = validators?.filter((validator) => {
+    return usersValidatorsMap?.has(validator.operator_address);
+  });
+
+  if (validators?.length && myValidators?.length) {
+    validatorBlock = (
+      <div className="flex flex-row space-x-2">
+        {myValidators?.slice(0, maxVisibleValidators).map((validator) => {
+          const imageUrl = queryValidators.getValidatorThumbnail(
+            validator.operator_address
+          );
+
+          return (
+            <div
+              className="h-10 w-10 overflow-hidden rounded-full"
+              key={validator?.description?.moniker}
+            >
+              <img alt={validator?.description?.moniker} src={imageUrl || ""} />
+            </div>
+          );
+        })}
+
+        {myValidators?.length > maxVisibleValidators && (
+          <AvatarIcon
+            extraValidators={myValidators?.length - maxVisibleValidators}
+          />
+        )}
+      </div>
+    );
+  }
+
+  return (
+    <>
+      <div className="mx-2 flex items-center">
+        <span className="caption text-sm text-osmoverse-200 md:text-xs">
+          {t("stake.validatorHeader")}
+        </span>
+        <div className="pl-5">
+          <Button
+            size="xs"
+            mode="bullish-special"
+            onClick={() => {
+              setShowValidatorModal(true);
+            }}
+          >
+            {t("stake.edit")}
+          </Button>
+        </div>
+      </div>
+      <OsmoverseCard>
```

Comment:
@MaxMillington let's inline the OsmoverseCard with it's styles. One of the main benefits of Tailwind is being able to see the styles at face value along with the markup. Otherwise, we're emulating styled components / Chakra UI via Tailwind, which comes with it's shortcomings of blending actual components which offer deep functionality and just styles which are static.